src/tilemap/TileMap.cs

```diff
@@ -171,5 +188,20 @@ public void Save()
         {
             throw new NotImplementedException();
         }
+
+        public void AddTileSet(TileSet ts) {
+            nameToSet.Add(ts.Name, ts);
+        }
+
+        //given a name, a TileSet with that name will be returned.
+        public TileSet NameToSet(String name) {
+            
```

Comment:
Avoid random spaces if possible. (As in it looks off or not aesthetically pleasing).

Reply:
What spaces exactly?